### PR TITLE
Microbiology plugin for seqstore transfer

### PR DESCRIPTION
**Pull request by JPWerling**
Plugin for collecting and transferring relevant data and files to seqstore from the IonTorrent S5 machines.

### CODE REVIEW

File: microbiology-plugin/covid_seqstore_transfer/covid_seqstore_transfer.py [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+        sample_name.replace(character, '')
+
+    for character in ['_']:
+        sample_name.replace(character, '-')
+
+    return sample_name
+
+
+def is_covid_sample(sample_name):
+    return re.search('D[A-Z]2[0-9]-[0-9]+', sample_name, re.IGNORECASE)
```
Comment (pbiology) [inline microbiology-plugin/covid_seqstore_transfer/covid_seqstore_transfer.py:85]: Do we need to confirm with micro that samples always will start with D?

File: microbiology-plugin/covid_seqstore_transfer/covid_seqstore_transfer.py [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+    for sample_output in sample_outputs:
+        barcode = os.path.basename(sample_output)
+        vcf_path = os.path.join(sample_output, 'TSVC_variants_{}.vcf.gz'.format(barcode))
+        vcf_collection[barcode] = vcf_path
+
+    return vcf_collection
+
+
+class covid_seqstore_transfer(IonPlugin):
+    version = "0.1.0.0"
```
Comment (pbiology) [inline microbiology-plugin/covid_seqstore_transfer/covid_seqstore_transfer.py:150]: hardcoded versions?

**Review by pbiology — APPROVED** [review]

File: microbiology-plugin/covid_seqstore_transfer/covid_seqstore_transfer.py (hunk 1 above)
Comment (JPWerling) [reply microbiology-plugin/covid_seqstore_transfer/covid_seqstore_transfer.py:85]: Probably. Fairly sure we asked a long time ago about the naming, but it's probably best to ask again.

File: microbiology-plugin/covid_seqstore_transfer/covid_seqstore_transfer.py (hunk 2 above)
Comment (JPWerling) [reply microbiology-plugin/covid_seqstore_transfer/covid_seqstore_transfer.py:150]: A requirement by the torrentserver. It reads plugin version directly from here, so you can set the version to whatever you like as long as it increments.

**Pull request merged**